composer.json

```diff
@@ -61,6 +61,10 @@
         "psr/container-implementation": "1.0.0"
     },
     "extra": {
+        "branch-alias": {
```

Comment:
This commit needs deleting from this PR.

Reply:
I have no idea how it got in there. Dropped from PR